src/test/scala/io/moia/protos/teleproto/HierarchicalProtocolBuffersTest.scala

```diff
@@ -88,6 +90,7 @@ class HierarchicalProtocolBuffersTest extends UnitTest {
 
     "use an 'explicit' implicit writer before generating a writer for a type in hierarchy of a generated type pair" in {
 
+      @nowarn // unused
```

Comment:
The compiler regards this as unused but actually it's the _explicit_ writer under test. Thus the annotation.

Reply:
I think:
```suggestion
      @nowarn("cat=unused")
```

Reply:
I didn't know that. Changed in https://github.com/moia-dev/teleproto/pull/97/commits/14362b1a8b9a10d34c3bd8e8595c0ba2254db926.